cmd/list/breakglasscredential/cmd.go

```diff
@@ -66,7 +66,7 @@ func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command) error {
 	}
 
 	if len(breakGlassCredentials) == 0 {
-		r.Reporter.Infof("there are no break glass credentials for this cluster")
+		r.Reporter.Infof("there are no break glass credentials for cluster '%s'", clusterKey)
```

Comment:
Super nit: `there` is capitalized in the external auth info message. Maybe you should choose one style to be consistent?